docker-compose.yml

```diff
@@ -38,6 +38,7 @@ services:
       - "127.0.0.1:6379:6379"
   mongo:
     image: circleci/mongo:3.6
+    platform: linux/amd64
```

Comment:
why do we need this?

Reply:
That image doesn't support ARM and Docker throws an error instead of defaulting to that platform if it's not explicitly specified.